### DISCUSSION TITLE
floating fungus bombs for aphotic plateau/hydrothermal wastes

### DISCUSSION THREAD

**Discussion by LePiromano** [FakeFishGames/Barotrauma#12380] [Ideas and suggestions]
you know how fungus caves have these fungus bombs sticking on the walls that deal damage when you get near them well kinda like that but they're just massive and floating all over the place in both aphotic plateau and hydrothermal wastes
im suggesting these because those 2 biomes are boring as all hell and everyone agrees they could use at least more than 1 enviromental hazard because all they have is ballast flora
the fungus bombs would be as big as a floating iceberg and act pretty much the same but they'd have less health and would straight up blow up and leave an acid cloud like watcher's when touched or destroyed
also everyone's been suggesting sea mines for years but they would make no sense story wise, however this would and is a cool concept (i think)

here's some concept design
![dgong](https://github.com/Regalis11/Barotrauma/assets/55637865/01e7b171-6989-4019-8500-af557f69ab58)
![dgong2](https://github.com/Regalis11/Barotrauma/assets/55637865/c5ac3d15-6950-4efb-b408-4644fd3a3a24)

representation video by uberpendragon
https://github.com/Regalis11/Barotrauma/assets/55637865/81633131-f4b7-4e83-a3ee-71d7f580db78

**Comment by ubertpendragon** [comment]
I tested it and honestly it's just iceberg 2.0

Looks cool except it's smaller and the AI wont react to it.
